v3/internal/runtime/desktop/@wailsio/runtime/src/index.ts

```
@@ -76,3 +76,21 @@ window._wails.handleDragLeave = handleDragLeave;
 window._wails.handleDragOver = handleDragOver;
 
 System.invoke("wails:runtime:ready");
+
+// Load optional window init scripts (fire and forget, matching current async behavior)
+// The backend identifies the window from the x-wails-window-id header
+fetch('/wails/init.js')
+    .then(r => r.ok && r.status !== 204 ? r.text() : null)
+    .then(js => { if (js) eval(js); })
```

Biome (2.1.2)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)